addons/my-ocular/profile.js

```diff
@@ -1,46 +1,30 @@
 export default async function ({ addon, console, msg }) {
-  const username = document.querySelector("#profile-data > div.box-head > div > h2").innerText?.split("#")[0]; // Fix bug with user-id addon
-  let container = document.querySelector(".location");
+  const username = Scratch.INIT_DATA.PROFILE.model.username;
   let data = await fetchStatus(username);
 
-  addon.settings.addEventListener("change", updateOcular);
-  addon.self.addEventListener("disabled", () => updateOcular(true));
-  addon.self.addEventListener("reenabled", () => updateOcular());
-
   if (typeof data.userStatus !== "string") return;
 
-  // Create status element
-  let statusSpan = document.createElement("span");
-  statusSpan.title = msg("status-hover");
-  statusSpan.innerText = data.userStatus;
-  statusSpan.style.fontStyle = "italic";
-  statusSpan.style.setProperty("display", "inline-block", "important");
-  statusSpan.id = "my-ocular-span";
+  const span = document.createElement("span");
+  span.textContent = data.userStatus;
+  span.className = "sa-ocular-status";
+
+  span.style.fontStyle = "italic";
+  span.style.display = "inline-block !important";
 
-  // Create my-ocular-dot
-  let dot = document.createElement("span");
+  const dot = document.createElement("span");
   dot.title = msg("status-hover");
-  dot.classList.add("my-ocular-dot");
-  dot.style.setProperty("display", "inline-block", "important"); // I have to do it like this because .style doesn't let me set prio, and featured project banner messes with this without !important
   dot.style.backgroundColor = data.color;
+  span.appendChild(dot);
 
-  // Create location element
-  let locationElem = document.createElement("span");
-  locationElem.id = "my-ocular-old-location";
-  locationElem.innerText = container.innerText; // Set it to the old innerText
-  locationElem.fontStyle = "italic";
-  container.innerText = ""; // Clear the old location
+  addon.tab.appendToSharedSpace({
+    space: "afterProfileCountry",
+    element: span,
+    order: 4, // set back due to width of element
```

Comment:
I'm going to change it to 9, in case there's more elements in the future.

```suggestion
    order: 9, // set back due to width of element
```